dockers/docker-dhcp-relay/dhcpv4-relay.agents.j2

```diff
@@ -24,6 +24,9 @@ command=/usr/sbin/dhcrelay -d -m discard -a %%h:%%p %%P --name-alias-map-file /t
 {% for (name, prefix) in PORTCHANNEL_INTERFACE|pfx_filter %}
 {% if prefix | ipv4 %} -iu {{ name }}{% endif -%}
 {% endfor %}
+{% for (name, gateway) in VLAN_INTERFACE|get_primary_addr %}
```

Comment:
Does this change the existing behavior if no secondary address is present? I thought we are setting `-pg` only if there is secondary address in the graph.

Reply:
We only add -pg for the VLANs which has secondary subnets present. So the current behavior of no secondary subnet present would not add any -pg parameters